ogr/services/forgejo/flag.py

```diff
@@ -20,11 +81,44 @@ def set(
         context: str,
         trim: bool = False,
     ) -> "ForgejoCommitFlag":
-        raise NotImplementedError("TBD")
 
-    @staticmethod
-    def get(
-        project: "ogr_forgejo.ForgejoProject",
-        commit: str,
-    ) -> Iterable["ForgejoCommitFlag"]:
-        raise NotImplementedError("TBD")
+        if isinstance(state, str):
+            state = super(ForgejoCommitFlag, ForgejoCommitFlag)._state_from_str(state)
+
+        state = ForgejoCommitFlag._validate_state(state)
+
+        if trim:
+            description = description[:140]
+
+        owner = project.forgejo_repo.owner.full_name
```

Comment:
Actually not sure, if `full_name` is what you want 👀 Though I admit there are at least 2 fields resembling a login and it's not very straightforward to guess which one is the one we want /o\